docs/guides.rst

```diff
@@ -0,0 +1,293 @@
+Guides and examples
+===================
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. _guide_gateway:
+
+Configuring an Internet Gateway
+-------------------------------
+
+An Internet gateway is a host which is managing the access of a private
+(internal) network to the (external) Internet. When running Linux as a gateway
+host the correct setup of the required iptables rules is crucial. While there
+exist dedicated Linux distributions for this task such as OpenWRT or IPFire,
+it's also possible to use a regular Debian system and configure it through
+DebOps. Here a short overview about the basic steps for a simple gateway
+configuration is given.
+
+The gateway host should have at least two network interfaces connected to the
+respective networks. In this guide the interface named ``eth0`` will be used
+as internal trusted interface and ``eth1`` will be used as external untrusted
```

Comment:
First off, I would switch `eth0` and `eth1` around. In Debian systems (pre-Stretch, I imagine) `eth0` is the default network interface, which is usually public. This makes sense when you add another interface, since if you move the default interface to `eth1`, your connection might be broken, and if not properly configured you will lose connectivity to the remote system.

In DebOps, `debops.ifupdown` role treats the `eth0` interface as an external, public interface, and `eth1` as an internal, private interface.

Furthermore, by default `debops.ifupdown` will configure two network bridges, `br0` attached to an external interface, and `br1` attached to an internal interface. This has dual purpose - it allows you to decouple the physical interfaces from logical, so that if you need to you can switch `eth1` to be an external and still keep the same `br0` interface name as external, and this layout makes it very easy to support virtual machines or containers with external and internal connectivity out of the box.